demo/config/targets/SYSTEM/lib/limits_groups.rb

```diff
@@ -0,0 +1,41 @@
+# encoding: ascii-8bit
+
+# Copyright 2014 Ball Aerospace & Technologies Corp.
+# All Rights Reserved.
+#
+# This program is free software; you can modify and/or redistribute it
+# under the terms of the GNU General Public License
+# as published by the Free Software Foundation; version 3 with
+# attribution addendums as found in the LICENSE.txt
+
+require 'cosmos/tools/cmd_tlm_server/limits_groups_background_task'
+
+module Cosmos
+  class LimitsGroups < LimitsGroupsBackgroundTask
+    def initialize(initial_delay)
+      super(initial_delay)
+      @temp1_enable_code = Proc.new do
+        #Logger.info "Enabling group GROUND"
+        enable_limits_group('GROUND')
```

Comment:
I noticed the logging when I didn't see the message for this enable_limits_group. That's why I think it should be in the API.